woof-code/rootfs-petbuilds/activate-linux/etc/xdg/autostart/activate-linux.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Version=1.0
+Name=activate-linux
+Comment=A full-featured GTK based trayicon UI for ConnMan
+Exec=sh -c '. /etc/DISTRO_SPECS; GDK_BACKEND=wayland exec activate-linux "$DISTRO_NAME $DISTRO_VERSION" "This copy of $DISTRO_NAME is not genuine"'
```

Comment:
what about "is a development build for testing" instead of "is not genuine"
Given that "genuine" puppy builds are not defined anywhere I can find, "not genuine" does not mean anything really and certainly does not mean that this is a development build for testing

Reply:
It's "genuine" as in https://www.google.com/search?q=windows+is+not+genuine+watermark&tbm=isch